### PR TITLE
Bump packages

### PR DESCRIPTION
**Pull request by thekp**
Resolves n/a

**Overall change:** _bumping psammead packages_

**Testing:**  Light Regression on changed packages required

---

- [x] I have assigned myself to this PR and the corresponding issues
- [n/a] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by jamesbhobbs — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -78,33 +78,33 @@
...
+    "@bbc/psammead-figure": "^1.2.3",
+    "@bbc/psammead-headings": "^3.1.3",
+    "@bbc/psammead-image": "^1.2.1",
+    "@bbc/psammead-image-placeholder": "^1.2.3",
+    "@bbc/psammead-inline-link": "^1.3.2",
+    "@bbc/psammead-locales": "^1.3.2",
+    "@bbc/psammead-media-indicator": "^2.5.3",
+    "@bbc/psammead-navigation": "^2.2.3",
+    "@bbc/psammead-paragraph": "^2.2.3",
+    "@bbc/psammead-section-label": "^2.3.3",
```
Comment (andrew-nowak) [inline package.json]: please pin to 2.1.6 - there's gotchas in 2.2.0 that I am addressing in #3266 :)

**Review by andrew-nowak — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -1126,213 +1126,198 @@
...
-        "@bbc/psammead-styles": "^2.0.2"
+        "@bbc/gel-foundations": "^3.3.2",
+        "@bbc/psammead-styles": "^2.1.1"
       }
     },
     "@bbc/psammead-section-label": {
-      "version": "2.1.5",
-      "resolved": "https://registry.npmjs.org/@bbc/psammead-section-label/-/psammead-section-label-2.1.5.tgz",
-      "integrity": "sha512-ffdpDFlraL+eBdYkmneZmyzkusWi06vM7rj2xb3jSgWw2Ntlw+rBQOnHqZAePI84iRbHb9eZ3ll0A9uZ3hT2fA==",
+      "version": "2.3.3",
```
Comment (andrew-nowak) [inline package-lock.json]: you might need to remove the caret from the package.json; it still seems to have picked >2.1.6 :(

**Review by andrew-nowak — CHANGES_REQUESTED** [review]

**Review by andrew-nowak — APPROVED** [review]
🙌 lgtm, thanks @thekp

**Comment by PriyaKR** [comment]
Regression looks good to me.

**Pull request merged**